### DISCUSSION TITLE
filter suggestions for the future

### DISCUSSION THREAD

**Discussion by Quikkby** [keiwando/toonsquid#37] [Ideas]
making a list of filters
I dont know how easy it is to make, but I beleive in you guys, coding is hard! (seriously! you guys work fast! but make sure to not get burned out <3)

what i do reccomend how to implement it, is making it so the edit frame drawing options, have a filter option there, so  you dont have to make a new layer every time you wanna change like the color of the border or sum and can change it in the current frame setting
also make it so it doesnt add onto the frame, and instead be seperate, and easily can be taken off if not needed anymore 

along with optacity slider for the effects too!

Blur/lensblur/motionblur
glow (option to chance color or use the drawing's color)
border glow (that you can change color)
border (that you can change color)
noise filter (size/intensity/color slider. generates on the whole canvas, but can be clipped/masked onto layers)
inner glow (that you can change color)
directional lighting (in Adobe animate, they have this option i think, basically takes one side of the drawing and makes it a color, mostly a glow, and the other side (optional) a color too, please add optacity slider for both the directions too, it doesnt have to be fancy and advanced, but ill show a example)
![image](https://user-images.githubusercontent.com/92064767/198409915-286c0717-986a-4737-8942-fcccddc99d74.png)
dropshadow (can change color, not sure if this already exists)

moving effects
shaker (drawing, will just make it shake as long as the duration of the frame.. shake based on intensity and rotation sliders)

harder ones that may need a more advanced team members to complete, may be in the later future
shaker (vector, boils line effect (if you know the boiler adobe animate plugin) basically just makes the lines jittery

i also reccomend looking at ibis paint x filters if you want to add any more from those, they have alot of filters for their raster drawing app

thank you!
any questions about my wording will be answered, if youre confused

**Comment by keiwando** [comment]
Thank you for the detailed suggestions! I will definitely come back to this thread once I start working on filters / effects.
